sim/firesim-lib/src/main/cc/bridges/blockdev.h

```diff
@@ -14,76 +14,78 @@
 #define MAX_REQ_LEN 16
 
 struct blkdev_request {
-    bool write;
-    uint32_t offset;
-    uint32_t len;
-    uint32_t tag;
+  bool write;
+  uint32_t offset;
+  uint32_t len;
+  uint32_t tag;
```

Comment:
I strongly disagree with per-project policies instead of language-specific policies. For C++, 2 spaces and 80 characters are quite common and enforced in other related projects. I find Scala very hard to read since it exceeds 100 characters per line (which I think the limit might be in FireSim). Keeping indentation to 2 spaces and line length to 80 characters results, in my opinion, in a better overview of the code.

Reply:
My suggestion here would be to not necessarily condition one on the other. C++ can have a significant rightward drift if lambdas are used excessively, which would suggest 2 spaces of indentation. But I could also see other languages having 4 spaces as the idiomatic indentation, which is totally fine. And you might have languages like Make that rely on tabs, so unifying this across language boundaries seems like it's more trouble than benefit. We should definitely stay consistent within languages though.

Reply:
@davidbiancolin confirmed that the scala is using indent of 2 spaces.  so the indenting comment I made here is moot.  I was primarily concerned about Scala and C++ because it is most likely that one would be looking at both those at the same time in this project.  I should have been more precise and limited in my comment.

I'm fine with shifting it to 2 spaces.  I used to like 2 spaces more 20 years ago. As my eyes have gotten older, I like it less. I should probably suck it up and increase the font size on my display again or go to the eye doctor. 🤣 

Excessive use of lambdas should probably be avoided, as should signatures that take a page of code to represent without binpacking the arguments (or 10 lines with them packed...).

David also mentioned that the Scala column limit is currently 120.  That seems excessive to me but the coming adoption of scalafmt will make it easier to reflow that on the fly.